### DISCUSSION TITLE
Default Playback Speed

### DISCUSSION THREAD

**Discussion by DenisSlobodyanyuk** [iina/iina#3919] [General]
Hi. I searched the web and preferences and I have not been able to find an option to set the default playback speed. Is it possible to do that with IINA? It seems like a better player than VLC but this feature is a deal breaker. Thanks

**Comment by low-batt** [comment]
IINA is a graphical user interface for the [mpv](https://mpv.io/) player. The mpv player is intended to be used from the command line and does not provide a GUI, only a small controller for basic control. IINA is one of several projects that provide frontends for mpv.

Many of the features of mpv that IINA does not provide a GUI for are still available to you using `Advanced` preferences. From the mpv player manual section [option speed](https://mpv.io/manual/stable/#options-speed):
>**--speed=<0.01-100>**
>
>    Slow down or speed up playback by the factor given as parameter.
>
>    If --audio-pitch-correction (on by default) is used, playing with a speed higher than normal automatically inserts the scaletempo2 audio filter.

Follow these steps:
- Start IINA
- Click on  `Preferences...` under the `IINA` menu
- Click on `Advanced`
- Slide the `Enable advanced settings` toggle button to be on (blue)
- In the `Additional mpv options` section click on `+`
- A new entry appears in the table
- Double click on `name` and replace it with `speed`
- Double click on `value` and replace it with `4.5`
- Confirm the panel looks like the screenshot below
- **Restart IINA** to active the new settings
![discussion-3919](https://user-images.githubusercontent.com/86170219/185733269-2a83a334-a780-4223-ad1a-a5af5f79947e.jpg)


With this configuration IINA will default to 4.5 as the playback speed.

**Reply by DenisSlobodyanyuk** [reply]
This worked for me. Many thanks.
